gemfiles/rails_52.gemfile

```diff
@@ -1,9 +1,9 @@
 # This file was generated by Appraisal
```

Comment:
FYI: This is auto-generated files and nobody should change them (i.e. change double quotes to single quotes).